src/rpc/net.cpp

```diff
@@ -125,7 +125,7 @@ static RPCHelpMan getpeerinfo()
                     {RPCResult::Type::NUM, "timeoffset", "The time offset in seconds"},
                     {RPCResult::Type::NUM, "pingtime", /*optional=*/true, "ping time (if available)"},
                     {RPCResult::Type::NUM, "minping", /*optional=*/true, "minimum observed ping time (if any at all)"},
-                    {RPCResult::Type::NUM, "pingwait", /*optional=*/true, "ping wait (if non-zero)"},
+                    {RPCResult::Type::NUM, "pingwait", /*optional=*/true, "The duration in milliseconds (ms) of an outstanding ping (if non-zero)"},
```

Comment:
In 3c196b1c1dacc0c1d67d6c3ebd756e2614fef89f:

Doc change seems fine, but isn't a bugfix/backport right? Any reason to change just this field? i.e why not make the same change to `pingtime`?

Reply:
> In [3c196b1](https://github.com/bitcoin/bitcoin/commit/3c196b1c1dacc0c1d67d6c3ebd756e2614fef89f):
> 
> Doc change seems fine, but isn't a bugfix/backport right? Any reason to change just this field? i.e why not make the same change to `pingtime`?

Good idea, will look at doing that. Agree, not a backport candidate.

Reply:
Done as well for pingtime and minping in the same commit.